### DISCUSSION TITLE
[Bug] How to distinguish the click and right-click command

### DISCUSSION THREAD

**Discussion by JackieCheung** [dream-num/univer#2263] [Q&A | 常见问题]
### Before you submit this issue, have you checked the following

- [X] Is this really a problem?
- [X] I have searched the Github Issues for similar issues, but did not find anything.

### Affected packages and versions

0.1.10

### Reproduction link

click or right-click, the command sheet.operation.set-selections will be triggered
how to distinguish whether it is a click or right-click

### Expected behavior

distinguish whether it is a click or right-click

### Actual behavior

cannot distinguish whether it is a click or right-click

**Comment by wzhudev** [comment]
Hello. `sheet.operation.set-selections` may be triggered by other behavior like keyboard navigation and switching worksheets. If you want to handle user interaction, it is not suggested to use this operation.
